src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -393,6 +393,9 @@ public override BoundNode VisitObjectOrCollectionValuePlaceholder(BoundObjectOrC
         public override BoundNode VisitInterpolatedStringArgumentPlaceholder(BoundInterpolatedStringArgumentPlaceholder node)
             => PlaceholderReplacement(node);
 
+        public override BoundNode? VisitInterpolatedStringHandlerPlaceholder(BoundInterpolatedStringHandlerPlaceholder node)
```

Comment:
nit: we should similarly add a check in `LocalRewritingValidator` below for `BoundInterpolatedStringHandlerPlaceholder`.